vue-components/src/components/OptionsMenu.vue

```diff
@@ -3,7 +3,9 @@
 		:class="[ 'wikit', 'wikit-OptionsMenu' ]"
 		@scroll.passive="onScroll"
 		ref="lookup-menu"
+		role="listbox"
 		:style="{ maxHeight: maxHeight ? maxHeight + 'px' : null }"
+		aria-label="Options Menu"
```

Comment:
Not sure which label to put here... Put this in the meantime. Not having an `aria-label` in the OptionsMenu generates an accesibility error.

Reply:
In storybook in the OptionsMenu story you mean? I think those lack the context of the surrounding lookup and so cause this false-positive error. As long as we use that component in a Lookup or dialog, we can probably ignore it.

Reply:
Added the label from the lookup as agreed.